importer/importer.go

```diff
@@ -30,9 +39,11 @@ func (asi AccountingServiceImpl) Process(stream goat_grpc.AccountingService_Proc
 		return err
 	}
 
-	clientID := id.GetIdentifier()
-	if clientID == "" {
-		return errors.New("First message in the stream must be client identifier")
+	switch id.Data.(type) {
```

Comment:
I really wanted to write `if id.Data.(type) != *goat_grpc.AccountingData_Identifier`, but Go won't let me use `.(type)` outside of a "type switch". Yes, that is a thing unfortunately :disappointed:

Reply:
:joy: Oh jeez.